Explorer/Assets/DCL/AvatarRendering/AvatarShape/Systems/AvatarShapeVisibilitySystem.cs

```diff
@@ -13,7 +13,7 @@ namespace DCL.AvatarRendering.AvatarShape.Systems
     [UpdateInGroup(typeof(CameraGroup))]
     public partial class AvatarShapeVisibilitySystem : BaseUnityLoopSystem
     {
-        private const float AVATAR_MINIMUM_CAMERA_DISTANCE_SQR = 1;
+        private const float AVATAR_MINIMUM_CAMERA_DISTANCE_SQR = 3.5f * 3.5f;
```

Comment:
I know it's a `const` and will be calculated once, but is there a reason not to have the final value and have it written like this?

Reply:
For me it's easier to know we mean "3.5 meters" otherwise if I put 12.25 I have to go and make the calculation to know what actual distance we are talking about.

Reply:
I see, makes perfect sense! Thanks 😋